docs/js/i18n.md

```diff
@@ -98,8 +98,28 @@ Then enter the two character i18n standard language specifier (e.g. "fr", "de",
 
 ## Removing i18n and react-intl
 
-You can remove `react-intl` modules by first removing the `IntlProvider` object from the `app/app.js` file and by either removing or not selecting the i18n text option during component scaffolding.
+You can remove `il8n` and `react-intl` by:
 
-The packages associated with `react-intl` are:
+### 1. Removing the following files:
+ - `app/i18n.js`
+ - `app/containers/LanguageProvider/`
+ - `app/translations/`
+ - `app/tests/i18n.test.js`
+
+### 2. Editing the following files:
+
+`app/app.js`
+ - Remove all references to `il18n` and `LanguageProvider`
```

Comment:
@kevinnorris Do you think it'd be better to mention the exact lines (with possibly line number)? See [this](https://help.github.com/articles/getting-permanent-links-to-files/) and [this](https://stackoverflow.com/questions/23821235/how-to-link-to-specific-line-number-on-github) to see how Github creates permanent links for referencing.

For eg. 
1. Remove `import LanguageProvider from 'containers/LanguageProvider';`
2. Remove `<LanguageProvider />` wrapper around `<Router />`.

So that there's no confusion. Generally devs are wary of touching boilerplate code. I'm only suggesting this because there are a lot of references to delete.

Reply:
@KarandikarMihir good call

@kevinnorris thank you! also, minor typo:

```
Remove all references to `il18n` and `LanguageProvider`
                            ^ i18n
```